src/Input/RadioButton/RadioButton.tsx

```diff
@@ -5,26 +5,35 @@ import classNames from 'classnames';
 import { RadioContainer, RadioLabel } from '../../Style/Input/RadioButtonStyle';
 
 const RadioButton: React.FunctionComponent<Props> = props => {
-  const { label, theme, className, ...defaultProps } = props;
```

Comment:
No usage of `theme` in our codebases and there is only one color in GDS for this variant.